031-DevOpsWithGitHub/Student/challenge01.md

```diff
@@ -1,68 +1,54 @@
+Now that we have a basic understanding of version control and Git, lets get some code checked into source control. DevOps best practices can apply to any programming language, so for today we have provided you a simple .NET Core web application to use.
 
-Review the following introduction to GitHub project boards: [GitHub project boards](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).
+If we were not supplied a repository in the next few steps you will need to create a repository and commit the supplied hackathon resource files into your repository supplied by your coach.
 
-### Challenge
+1. Create a new repo for the hackathon. This is where you work during the event
 
-1. Add collaborators from your team to your repo. 
+2. Begin by cloning the GitHub repository you created in the [first challenge](challenge01.md) to your local computer ([hint](https://help.github.com/en/articles/cloning-a-repository)).
 
-2. Create a new project using the "Team backlog" template.
+3. Next, obtain the code (sample application and ARM template) from the Resource files provided by your coach.
 
-3. Ensure your project is with linked with your repository
+3. Finally, commit the files to your GitHub repository using your preferred Git client.
 
-4. Create a draft issue for this challenge and each of the remaining challenges (i.e., create 9 issues total - skip challenge 0). When creating each issue, make sure to:
-    - Assign an owner for each issue from your team. 
-    - Convert the `challenge 1` draft issue to an issue and select our repository. Look at the difference between a draft and your converted `challenge 1` issue
+### Challenge (if supplied a repo - your coach will tell you the URL and add you to this repo as a contributor)
 
-5. Move all your issues to the "Ready" column and convert each one from draft to an issue.
+Now that we have a basic understanding of version control and Git, lets get some code checked into source control. DevOps best practices can apply to any programming language, so for today we have provided you a simple .NET Core web application to use.
 
-6. Since you are currently working on Challenge 1, drag the `Challenge 1` issue to the "In progress" column.
-
-7. Ask your team member to close the `Challenge 1` issue you assigned to them. Review your project board to see if the issue moved to the 'Done' column automatically. 
-
-8. Create a new view to show your challenge issues in a table view. 
-
-9. Add a custom field to your issue. The custom field can be called Complexity and will be a single select type with the options (Very Easy, Easy, Hard and Very Hard).
-
-10. After you complete each of the following challenges, be sure to close each associated issue. 
+1. Begin by cloning the GitHub repository to your local computer ([hint](https://help.github.com/en/articles/cloning-a-repository)) from the repository supplied to you by your coach.
 
```

Comment:
I am still brainstorming how to best convey the two different challenge approaches but can you confirm there should be 2 additional steps here similar to above?  I would have expected the challenges to mention obtaining the code samples and checking it in.

Reply:
When you are supplied a repo then I am making the assumption that the repo will already contain the resource files. Its a way to avoid having to pass the students these files by some other means. The setup script in our case does this for all the participants/teams.